src/spatio_temporal_voxel_grid.cpp

```diff
@@ -330,12 +341,38 @@ double SpatioTemporalVoxelGrid::GetFrustumAcceleration( \
 }
 
 /*****************************************************************************/
-void SpatioTemporalVoxelGrid::GetOccupancyPointCloud( \
-                                       pcl::PointCloud<pcl::PointXYZ>::Ptr& pc)
+void SpatioTemporalVoxelGrid::GetOccupancyPointCloud(\
+                                       sensor_msgs::PointCloud2::Ptr& pc2)
```

Comment:
Please align this with the guard  like before

Reply:
Ok